awx/api/filters.py

```diff
@@ -146,7 +146,7 @@ class FieldLookupBackend(BaseFilterBackend):
 
     # A list of fields that we know can be filtered on without the possiblity
     # of introducing duplicates
-    NO_DUPLICATES_WHITELIST = (CharField, IntegerField, BooleanField)
+    NO_DUPLICATES_ALLOW_LIST = (CharField, IntegerField, BooleanField)
```

Comment:
The intent of the naming was "filtering these field types will not introduce duplicates"

you could also call it `DUPLICATE_SAFE` or `DISTINCT_NOT_NEEDED`